demo/src/app/components/shared/api-docs/api-docs.component.html

```diff
@@ -53,5 +53,22 @@ <h3 id="outputs">Outputs</h3>
       </table>
     </section>
   </template>
+
+  <template [ngIf]="apiDocs.exportAs && apiDocs.methods.length">
+    <section>
+      <h3 id="methods">Methods</h3>
+      <table class="table table-sm table-hover">
+        <tbody>
+        <tr *ngFor="let method of apiDocs.methods">
+          <td class="col-md-3"><code>{{method.name}}</code></td>
+          <td class="col-md-9">
+            <div><i>Signature: </i><code>{{ _methodSignature(method) }}</code></div>
```

Comment:
Does ng2 support [tagged template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals#Tagged_template_literals)? I wonder if this would provide a more elegant implementation. I've no problem with it either way though.